fdbcli/fdbcli.actor.cpp

```diff
@@ -3239,6 +3243,59 @@ ACTOR Future<int> cli(CLIOptions opt, LineNoise* plinenoise) {
 					continue;
 				}
 
+				if (tokencmp(tokens[0], "suspend")) {
+					getTransaction(db, tr, options, intrans);
+					if (tokens.size() == 1) {
+						Standalone<RangeResultRef> kvs = wait( makeInterruptable( tr->getRange(KeyRangeRef(LiteralStringRef("\xff\xff/worker_interfaces"), LiteralStringRef("\xff\xff\xff")), 1) ) );
+						Reference<FlowLock> connectLock(new FlowLock(CLIENT_KNOBS->CLI_CONNECT_PARALLELISM));
+						std::vector<Future<Void>> addInterfs;
+						for( auto it : kvs ) {
+							addInterfs.push_back(addInterface(&address_interface, connectLock, it));
+						}
+						wait( waitForAll(addInterfs) );
+						if(address_interface.size() == 0) {
+							printf("\nNo addresses can be killed.\n");
+						} else if(address_interface.size() == 1) {
+							printf("\nThe following address can be killed:\n");
+						} else {
+							printf("\nThe following %zu addresses can be killed:\n", address_interface.size());
+						}
+						for( auto it : address_interface ) {
+							printf("%s\n", printable(it.first).c_str());
+						}
+						printf("\n");
+					} else if(tokens.size() == 2) {
+						printUsage(tokens[0]);
+						is_error = true;
+					} else {
+						for(int i = 2; i < tokens.size(); i++) {
+							if(!address_interface.count(tokens[i])) {
+								printf("ERROR: process `%s' not recognized.\n", printable(tokens[i]).c_str());
```

Comment:
It will reports error for the valid command `fdbcli suspend 100 ip1`, where ip1 is a worker's IP address.

I think the Line 3267 should be `if(tokens.size() <= 2)`

Reply:
> It will reports error for the valid command `fdbcli suspend 100 ip1`, where ip1 is a worker's IP address.

Why? In this case, `tokens` is `["suspend", "100", "ip1"]` and it should work just fine.

Reply:
When tokens size > 1, it will not set `address_interface `. Then `if(!address_interface.count(tokens[i])) ` will be true and reports process not recognized.

Interestingly, this is the same logic for `kill` command. 
It seems to use the command operator has to run `suspend` first before suspend any process.